litestar/datastructures/headers.py

```diff
@@ -272,7 +271,33 @@ def to_header(self, include_header_name: bool = False) -> str:
 
         return (f"{self.HEADER_NAME}: " if include_header_name else "") + self._get_header_value()
 
-
+    def dict(
```

Comment:
so, i dont see why we will need to replicate the pydantic signature here. Do we actually use `exclude_unset`, `exclude_empty` etc? Also, since we actually use this stuff internally - we do not need a generalized use here, but only what we actually use.